### DISCUSSION TITLE
DLL signing?

### DISCUSSION THREAD

**Discussion by rruprai1** [CommunityToolkit/Windows#358] [Q&A] [answered]
General question, mostly I don't know what the stand is with this project.

Why are the DLLs not signed?

I am  just looking for a explanation to further my understanding. For context, I work in a environment where unsigned dlls are frowned upon and used as an example of a possible attack vector, as the DLLs and its dependencies can't be trusted or there is not proof the DLL has not been modified.

**Comment by michael-hawker** [comment, marked as answer]
Hi @rruprai1 the packages we release to NuGet (and the dlls included) are all signed by the .NET Foundation, as I can see from my NuGet cache, for example, here:

![image](https://github.com/CommunityToolkit/Windows/assets/24302614/9c0695fc-caef-456b-8930-76c51ec856af)

![image](https://github.com/CommunityToolkit/Windows/assets/24302614/d3e0ca28-5573-4f35-98f4-fc4919b1e21a)

Are you using preview packages of ours? The ones from integrations with the `main` branch should also go through this same signing process. Otherwise, we're going to need more information about what you're seeing or concern is.
